clowdapp.yaml

```diff
@@ -190,11 +190,34 @@ objects:
           - name: SSL_CERT_DIR
             value: ${SSL_CERT_DIR}
           - name: SERVICE_NAME
-            value: "rosocp-housekeeper"
+            value: "rosocp-housekeeper-sources"
           - name: CW_LOG_STREAM_NAME
             value: "rosocp-housekeeper"
           - name: LOG_LEVEL
             value: ${LOG_LEVEL}
+
+    jobs:
+      - name: delete-rosocp-partitions
+        schedule: "0 0 */15 * *" # Runs at 12:00 AM, every 15 days.
```

Comment:
Can we keep schedule value configurable?

Reply:
Done